app/lib/foreman_remote_execution/renderer/scope/input.rb

```diff
@@ -23,6 +42,16 @@ def cached(key, &block)
           Rails.cache.fetch(cache_key, &block)
         end
 
+        # rubocop:disable Lint/InterpolationCheck
+        apipie :method, 'Render template by given name' do
+          required :template_name, String, desc: 'name of the template to render'
+          optional :input_values, Hash, desc: 'key:value list of input values for the template'
+          optional :options, Hash, desc: 'Additional options such as :with_foreign_input_set. Set to true if a foreign input set should be considered when rendering the template'
```

Comment:
@ares could you check if this makes sense? You're way more familiar with templates and their rendering than I am

Reply:
yup, that's correct